### PR TITLE
lightning v1.8.6

### PR DESCRIPTION
**Pull request by DManowitz**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Reset the build number to `0` (if the version changed)
* [X] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by DManowitz** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -23,40 +23,22 @@ requirements:
...
-    - dateutils <2.0
-    - deepdiff <8.0,>=5.7.0
-    - fastapi <2.0,>=0.92.0
-    - fsspec <2025.0,>=2022.5.0
-    - inquirer <5.0,>=2.10.0
-    - jinja2 <5.0
-    - lightning-cloud >=0.5.37
-    - lightning-utilities <2.0,>=0.7.0
-    - numpy <3.0,>=1.17.2
+    - arrow <1.2.4,>=1.2.0
```
Comment (Borda) [inline recipe/meta.yaml:26]: is it really all, seems to few regarding other is also Apps inside

Comment (weiji14) [reply recipe/meta.yaml:26]: These are the requirements I get from running `grayskull pypi lightning=1.8.6`:

```
requirements:
  host:
    - python >=3.7
    - pip
  run:
    - aiohttp >=3.8.0
    - arrow >=1.2.0
    - beautifulsoup4 >=4.8.0
    - click
    - croniter >=1.3.0,<1.4.0
    - deepdiff >=5.7.0
    - fsspec >2021.06.0
    - inquirer >=2.10.0
    - lightning-cloud >=0.5.12
    - lightning-utilities >=0.3.0,!=0.4.0
    - numpy >=1.17.2
    - packaging >=17.0
    - psutil
    - python >=3.7
    - pytorch >=1.9.0
    - pyyaml >=5.4
    - starsessions >=1.2.1,<2.0
    - tensorboardx >=2.2
    - traitlets >=5.3.0
    - torchmetrics >=0.7.0
    - tqdm >=4.57.0
    - typing-extensions >=4.0.0
```
It does look like there are a few packages missing. Can also cross-reference the requirements in the source tar.gz file at https://pypi.org/project/lightning/1.8.6/#files

**Comment by DManowitz** [comment]
As per the conversation [here](https://github.com/conda-forge/pytorch-lightning-feedstock/issues/134), I'm closing this PR.

**Pull request closed without merging**